source/_components/light.template.markdown

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: "Template Light"
+description: "Instructions how to integrate Template lights into Home Assistant."
+date: 2016-05-18 20:32
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_release: 0.46
+ha_iot_class: "Local Push"
+logo: home-assistant.png
+---
+
+The `template` platform creates lights that combine components.
+
+This provides pretty much the same functionality as a [template switch](/components/switch.template/) with the exception is that you can pass brightness commands through.
```

Comment:
I'd rewrite this paragraph assuming the user hasn't yet read about template switches. Just describe what this platform will do here.